admin/class-admin-apple-news.php

```diff
@@ -180,6 +180,19 @@ public function __construct() {
 				apple_news_register_meta_helper( 'post', $post_types, $meta_key, $options );
 			}
 
+			// Prevent Yoast Duplicate Post plugin from cloning apple_news meta.
+			add_filter(
+				'duplicate_post_meta_keys_filter',
+				function( $meta_keys ) {
+					array_filter(
```

Comment:
Filters should always return a value, so this should be `return array_filter` ... etc

Reply:
Also we should check to ensure `$meta_keys` is an array before iterating on it. It _should_ be, but with filters, we can't trust the input, because it could have been improperly filtered by other code before. So maybe do something like, `return is_array( $meta_keys) ? array_filter( /* logic here */ ) : $meta_keys` so the passed value is returned unmodified if it's not an array.